.github/workflows/scan-lint.yaml

```diff
@@ -17,8 +17,15 @@ jobs:
       - name: Install goimports
         run: go install golang.org/x/tools/cmd/goimports@latest
 
-      - name: Install golangci-lint
-        run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.55.2
+      - name: set golangci-lint version # keep in sync with src/tools/go.mod
```

Comment:
Instead of having separate job for golangci-lint, I'd like to include it as part of our pre-commit hook, which is run below (noting this is the same pre-commit hook that gets run when a dev makes a local commit). See pre-commit config [here](https://github.com/defenseunicorns/uds-cli/blob/5af5d20475863680820b0f4afe59cc44fca87c6a/.pre-commit-config.yaml#L24)

Reply:
Thanks, Is there a reason? I am trying to understand. 

I picked the scan-lint because it belonged to the scan-and-lint configuration, in my opinion.

Reply:
The scan-lint job is correct. Note that scan-lint calls the pre-commit hook, which calls `golangci-lint`, so no need to have a separate golangci-lint job in this workflow

Reply:
I understood. I appreciate the explanation. Thank you!